src/main/java/Steps/User.java

```diff
@@ -1,6 +1,6 @@
 package Steps;
 
-public class Post {
```

Comment:
why the name of class was post.
It's better ro use something that explains what entity is it.
SO User is the best one.